test/helpers.js

```diff
@@ -0,0 +1,30 @@
+// Helper function to determine whether this browser is supported by 
+// impress.js or not. Copied from impress.js itself.
+var _impressSupported = function() {
```

Comment:
Actually, I agree with your last suggestion. However, in this first commit, I didn't yet want to touch js/impress.js, since the religious belief is we must add tests before touching it. And I think it's cood to be overly conservative in these first steps. So I have copy pasted the lines I needed into this helpers file. In the future, I agree this code could be removed and instead we expose this functionality from impress.js itself.